src/Microsoft.AspNet.Server.Kestrel/Networking/Libuv.cs

```diff
@@ -12,12 +12,7 @@ public Libuv()
         {
             IsWindows = PlatformApis.IsWindows;
 
-            var isDarwinMono =
-#if DNX451
-                IsWindows ? false : PlatformApis.IsDarwin;
-#else
-                false;
-#endif
+            var isDarwinMono = !IsWindows && PlatformApis.IsDarwin;
```

Comment:
Whoops, forgot about this one. Yes, I'll look into it right not.

/cc @davidfowl @halter73